sky/shell/ui/engine.h

```diff
@@ -36,11 +37,10 @@ class Engine : public UIDelegate,
     Config();
     ~Config();
 
-    RasterCallback raster_callback;
     scoped_refptr<base::SingleThreadTaskRunner> gpu_task_runner;
```

Comment:
It is still used by `Engine::OnOutputSurfaceCreated` and `Engine::OnOutputSurfaceDestroyed`.